### DISCUSSION TITLE
fluent ui react card deprecated, what next?

### DISCUSSION THREAD

**Discussion by imir-hoxha** [microsoft/fluentui#23796] [Q&A] [answered]
It is really confusing. What do I have to use now that react card is deprecated?

**Comment by khmakoto** [comment, marked as answer]
`@fluentui/react-card` is not deprecated, it actually has a preview of a still unstable `Card` component that you are free to use at your own risk.

There is a separate package, `@fluentui/react-cards`, that has been deprecated and is not being worked on anymore.
